app/js/controllers.js

```diff
@@ -4,11 +4,22 @@
 'use strict';
 
 /* Controllers */
-
 function MainCtrl($scope, $timeout) {
   $scope.logoState = '';
   $scope.errorMessage = null;
   $scope.message = null;
+	$scope.decks = {
```

Comment:
Nitpick, inconsistent indentation throughout PR. Might be good to put an `.editorconfig` in for two-space indents to avoid future drift.